src/app/page.js

```diff
@@ -21,11 +21,20 @@ const sortOptions = [
 
 export default function Page() {
   const params = useSearchParams();
+  const router = useRouter();
   const alert = params.get("alert");
   const message = params.get("message");
   const [keyword, setKeyword] = useState("");
   const [sort, setSort] = useState(sortOptions[0].value);
 
+  useEffect(() => {
+    if (alert) {
+      setTimeout(() => {
+        router.push("/", { scroll: false });
+      }, 4000);
```

Comment:
This now can be simplified also

```suggestion
      setTimeout(() => router.push("/", { scroll: false }), 4000);
```